src/files/pyfingerprint/pyfingerprint.py

```diff
@@ -1420,7 +1421,7 @@ def uploadCharacteristics(self, charBufferNumber = FINGERPRINT_CHARBUFFER1, char
             raise Exception('Unknown error '+ hex(receivedPacketPayload[0]))
 
         ## Upload data packets
-        packetNbr = len(characteristicsData) / maxPacketSize
+        packetNbr = math.ceil(len(characteristicsData) / float(maxPacketSize))
```

Comment:
I am pretty sure that a cast to `int` would be sufficient here.

Reply:
Will be fixed in #106